.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: python
+python:
+    - "2.7"
+before_install:
+    - "export DISPLAY=:99"
+    - "sh -e /etc/init.d/xvfb start"
+    - "git fetch origin master:pylint-basis"
+    - "git diff pylint-basis --name-only | grep \\\\.py$ > CHANGED_PY_FILES || echo No changed .py files"
```

Comment:
Cool, I thought there might be a way to do that. It was tricky to figure out a script that would work correctly in all cases (i.e. some .py files changed or no .py files changed), without propagating an error exit code.